content/docs/0.8.0/install/requirements.md

```diff
@@ -5,7 +5,7 @@ weight: 1
 
 -  A container runtime compatible with Kubernetes (Docker v1.13+, containerd v1.3.7+, etc.)
 -  Kubernetes v1.14+.
--  `open-iscsi` has been installed on all the nodes of the Kubernetes cluster, and `iscsid` daemon is running on all the nodes.
+-  `open-iscsi` has been installed on all the nodes of the Kubernetes cluster, and `iscsid` daemon is running on all the nodes. This is necessary, as all longhorn volumes are created as iscsi targets under the hood.
```

Comment:
How about
```
Since Longhorn relies on `iscsiadm` on the host to provide persistent volumes to Kubernetes.
```

Also, we can update the latest doc (v1.0.2, instead of v0.8.0) and it will be picked up by the upcoming v1.1.0 release as well.

Reply:
off, not sure how I got to 0.8.0 only. I updated all docs according to your suggestion, PTAL.